### ISSUE TITLE
"Fatal error in launcher " occurs when I use vizviewer.

### ISSUE THREAD

**Issue by mengyuan1073** [gaogaotiantian/viztracer#262]
The result.json file generated successfully by using viztracer.  But when I try to visualize the file by command ''vizviewer result.json'' or
''vizviewer -h'' in Windows PowerShell,it can not work:

Fatal error in launcher: Unable to create process using '"D:\bld\viztracer_1652447918802\_h_env\python.exe"  "D:\anaconda\envs\python37\Scripts\vizviewer.exe" result.json': ???????????

Fatal error in launcher: Unable to create process using '"D:\bld\viztracer_1652447918802\_h_env\python.exe"  "D:\anaconda\envs\python37\Scripts\vizviewer.exe" -h': ???????????

The python version is 3.7.

Thanks for answers.

**Comment by gaogaotiantian** [comment]
This seems like an environment setup issue, rather than a viztracer issue. The `???????????` seems very odd to me. You could try to use `python -m viztracer.viewer result.json`. If that works, then it's definitely an environment issue. Maybe something is messed up in conda.

**Comment by mengyuan1073** [comment]
Using the command you provide, the problem is resolved. Much appreciation.

**Comment by randomseed42** [comment]
Met the same problem using vizviewer, and track down to a SO link https://stackoverflow.com/a/61091235/11767737, so installed viztracer in two conda clean environments, one is using conda-forge, one is using pip, then compared the binary vizviewer and the diff is only the python path as below
![image](https://github.com/gaogaotiantian/viztracer/assets/50793718/eaad623f-8ab6-4cea-a152-b9553cd3ba82)

And look into conda base pkgs folder and it seems conda-build created the meta.yaml file is using some wrong python path:
![image](https://github.com/gaogaotiantian/viztracer/assets/50793718/d9b62002-1454-4979-be4b-b1be9ec8659b)

So basically this is a conda-build bug but didn't dive more in conda-build issues. While I just wonder why other pip packages (like flask) are not having similiar issue? Is there any setting in pyproject.toml influence the meta.yaml file generation?
